# DO NOT EDIT. RUN THE PYTHON FILE INSTEAD TO GENERATE THIS .yaml FILE 
traj_x: 1.0*sin(0.454545454545455*t) + 2.0*sin(0.909090909090909*t)
traj_y: 1.0*cos(0.454545454545455*t) - 2.0*cos(0.909090909090909*t)
traj_z: -0.6*sin(1.36363636363636*t) + 1.9
bbox: [0.7, 0.7, 0.7]